extensions/interactions/RatioExpressionInput/directives/ratio-expression-input-validation.service.spec.ts

```diff
@@ -263,6 +263,10 @@ describe('RatioExpressionInputValidationService', () => {
   });
 
   it('should catch undefined value for # terms', () => {
+    // This throws "Type 'undefined' is not assignable to type 'number'
```

Comment:
Should the particular test case be removed if undefined is not a valid value ?

Reply:
done